setup.py

```diff
@@ -16,6 +20,51 @@
 with open("README.md", "r") as f:
     readme = f.read()
 
+
+class DevelopCommand(develop):
```

Comment:
Huh, I didn't know this convention existed. I will read about it and use it in projectmesa/mesa's setup.py